jira-automation.py

```diff
@@ -240,7 +240,7 @@ def main(args):
 
     snyk_client = SnykClient(snyk_api_token)
     snyk_org = snyk_client.get_organization(snyk_org_id)
-    projects = snyk_org.projects.all()
+    projects = snyk_org.projects.get("570fd96f-09ad-488b-bb37-f822c34c973f")
```

Comment:
for this, it will be better to have a configuration file to map the project id with names, to be easier for us later to easily identify the project here.